### ISSUE TITLE
Unscalarized New Frontend fails to find variable during connect algorithm

### ISSUE THREAD

**Issue by kabdelhak** [OpenModelica/OpenModelica#10839]
### Description
The model `Modelica.Fluid.Examples.AST_BatchPlant.Test.TankWithEmptyingPipe1` does not run with `--newBackend`. It throws and error during frontend stage:
```
[/var/lib/jenkins/ws/OpenModelicaLibraryTestingWork/OpenModelica/OMCompiler/Compiler/NFFrontEnd/NFConnectEquations.mo:1081:5-1082:59:writable] Error: Internal error NFConnectEquations.lookupVarAttr could not find the variable tank1.topPorts[i].m_flow
```

It looks like the variable is not known, because it is indexed (?).

**Comment by casella** [comment]
@kabdelhak never mind the model builders, those can be run offline once and for all with the existing backend. What is important is being able to run the actual built models inside `ScalableTestGrids.Models` 😅

**Comment by perost** [comment]
#10908 fixes the reported issue, and #10909 fixes the next issue that showed up (the DAE can't represent some builtin attributes as arrays, which is mainly a cosmetic issue since the NB doesn't actually use the DAE).

With these fixes the model passes the frontend at least, but fails in the NB for some reason.
